src/Desktop/UIAutomation/CustomObjects/Registrar.cs

```diff
@@ -9,21 +14,59 @@ namespace Axe.Windows.Desktop.UIAutomation.CustomObjects
 {
     public class Registrar : IDisposable
     {
-        CUIAutomationRegistrar _uiaRegistrar = new CUIAutomationRegistrar();
+        private IUIAutomationRegistrar _uiaRegistrar;
+        private Action<int, ITypeConverter> _callback;
```

Comment:
Suggestion: Make this readonly, use a more descriptive name. This callback assigns or registers a converter to a property ID